stdlib/src/array.rs

```diff
@@ -645,20 +647,48 @@ mod array {
     }
 
     impl Constructor for PyArray {
+        type Args = FuncArgs;
```

Comment:
```suggestion
        type Args = (PyStrRef, FuncArgs);
```
maybe this will work.

looking for other `fn py_new` may give more idea.

Reply:
Thank you! This is a really good suggestion :D